js/wordlist.js

```diff
@@ -549,7 +549,7 @@ function createSelectors() {
 }
 
 /* major function for displaying the Wordlist panel of the Edictor */
-function showWLS(start)
+function showWLS(start, separator="\t")
```

Comment:
wait a second: you cannot handle defaults in this way, can you? This should be:
```js
function showWLS(start, separator) {
  if (typeof separator == 'undefined') {separator = "\t"}
...
}
```

Reply:
I remember people discussing that kind of defaults recently. I think the conclusion was that all reasonable browsers support it, Internet Explorer does not. Feel free to change it, it looked like it did work on my browsers.

Reply:
[Also, I had meant to push that commit only *after* you merging the pull request. Sorry.]